README.md

```diff
@@ -1103,6 +1103,43 @@ sudo lunatik run examples/filter/sni false   # runs the Lua kernel script
 sudo xdp-loader load -m skb <ifname> https.o # loads the XDP/eBPF program
 ```
 
+### dnsblock
+
+This is a lua program to block DNS from a blacklist. To run this program follow the instructions below.
+
+#### Usage
+
+1. Build the userspace program
```

Comment:
Please mind the other examples.. we are using one single shell listing and using comments (e.g., `# builds the userspace module`) to give instructions like this